extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -16,6 +16,13 @@
 import {BaseCarousel} from './base-carousel';
 import {Layout} from '../../../src/layout';
 
+/** @const */
+const DIR_FORWARD = 1;
```

Comment:
Since most likely operation is `Math.sign()`, I'd say direct literals are better than obvious constants.